### PR TITLE
web: add info tooltip to widgets column header

### PR DESCRIPTION
**Pull request by landism**
image

(will be dismissable with #4907)

Copy advice welcome!

### CODE REVIEW

File: web/src/OverviewTable.tsx [hunk 1]
```diff
@@ -532,6 +532,22 @@ const columnNameToInfoTooltip: {
...
+        <a
+          href={linkToTiltDocs(TiltDocsPage.CustomButtons)}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          custom buttons docs
+        </a>
+        .
+      </div>
+    </>
```
Comment (lizzthabet) [inline web/src/OverviewTable.tsx:549]: wdyt about 
```
Buttons can be added to resources to easily perform custom actions. See button docs.
```
? maybe? i don't think it needs a line break or needs to be wrapped in a `div`.

File: web/src/OverviewTable.tsx [hunk 2]
```diff
@@ -532,6 +532,22 @@ const columnNameToInfoTooltip: {
       .
     </>
   ),
+  Widgets: (
```
Comment (lizzthabet) [inline web/src/OverviewTable.tsx:535]: (maybe bikeshedding?) this tooltip text is making me wonder if this column should just be named `Custom actions`? i forgot what naming we originally talked about... but honestly, feel free to ignore this suggestion / thought.

**Review by lizzthabet — APPROVED** [review]
🎉 ℹ️

File: web/src/OverviewTable.tsx [hunk 3]
```diff
@@ -532,6 +532,22 @@ const columnNameToInfoTooltip: {
...
+    <>
+      <div>You can add custom buttons here to easily perform actions.</div>
+      <div>
+        See{" "}
```
Comment (milas) [inline web/src/OverviewTable.tsx:539]: I'm both surprised I've never seen this trick before and also wish I hadn't 🙈

**Review by milas — APPROVED** [review]

File: web/src/OverviewTable.tsx (hunk 2 above)
Comment (landism) [reply web/src/OverviewTable.tsx:535]: I'm not a fan of putting "custom" in the names of things, but haven't really had a good conversation about it and might be wrong. I'm happy to talk about it, but am also pretty sure it's not an urgent thing and we can change it later, so not going to block.

File: web/src/OverviewTable.tsx (hunk 1 above)
Comment (landism) [reply web/src/OverviewTable.tsx:546]: Thanks!

**Pull request merged**